hoot-core/src/main/cpp/hoot/core/schema/TagComparator.cpp

```diff
@@ -524,8 +526,13 @@ void TagComparator::mergeNames(Tags& t1, Tags& t2, Tags& result)
     }
     else if (result.contains(it2.key()))
     {
-      QStringList sl = Tags::split(it2.value());
-      altNames.insert(sl.begin(), sl.end());
+      const Qt::CaseSensitivity caseSensitivity =
+        _caseSensitive ? Qt::CaseSensitive : Qt::CaseInsensitive;
+      if (result[it2.key()].compare(it2.value(), caseSensitivity) != 0)
+      {
```

Comment:
Ok.  Didn't even think about enumerated tags.  I can try to exempt them from the case sensitivity check or make a separate case sensitive parameter for them....whichever makes more sense.